tests/integration/docker_utils/test_docker.py

```diff
@@ -1228,6 +1228,9 @@ def test_is_container_running(self, docker_client: ContainerClient, dummy_contai
         docker_client.restart_container(name)
         assert docker_client.is_container_running(name)
         docker_client.stop_container(name)
+        import time
+
+        time.sleep(5)
         assert not docker_client.is_container_running(name)
```

Comment:
nit: wouldn't a retry be a bit more stable for this? Or is this just some leftover from debugging?

Reply:
Whoops, sorry! That's a leftover which smuggled itself into the stage... Thanks for catching that!